File: 1-TwoSum/1-TwoSum.cpp
// Last updated: 22/04/2025 16:49:43
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res;
        bool etat = false;
        for (int i = 0; i <nums.size(); i++) {
            if (etat == false){
                for(int j =0; j < nums.size(); j++){
                    if(j != i){
                        if (nums[i] + nums[j] == target){
                            res.push_back(i);
                            res.push_back(j);
                            etat = true;
                            break;
                        }
                    }
                }
            }else{
                break;
            }        
        }
        return res;
    }
};

File: 1-TwoSum/1-TwoSum.py
# Last updated: 22/04/2025 16:47:45
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        res: List[int] = []
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i] + nums[j] == target:
                    res.append(i)
                    res.append(j)
                    return res
